pages/success-banner/index.js

```diff
@@ -22,11 +57,13 @@ const Banner = () => (
       </h1>
     </div>
     <div className="horizontal center">
-      <Button href="https://github.com/resin-io/etcher">
+      <Button href="https://github.com/resin-io/etcher"
+        label="star on Github">
```

Comment:
nitpick: should it be `Star` with a capital `S`? ;)  (as that's what the button says)